pkg/sandbox/sandbox.go

```diff
@@ -196,6 +226,36 @@ func (s *SandboxController) ReconcileUserCreation(userName string) (*SandboxUser
 	return s.GetKubeconfigPathForSpecificUser(toolchainApiUrl, compliantUsername, kubeconfigPath, userToken)
 }
 
+func (s *SandboxController) GetKubeconfigPathForSpecificUserStage(toolchainApiUrl string, userName string, kubeconfigPath string, keycloakAuth *KeycloakAuth) (*SandboxUserAuthInfo, error) {
```

Comment:
this one is almost identical to `GetKubeconfigPathForSpecificUser` - would it make sense to unify functions/methods that are borrowing a major portion of code from each other?